test/mocha/10-restrictions.js

```diff
@@ -154,8 +154,41 @@ describe('restrictions', function() {
       secondRestriction.restriction.methodOptions.duration.should.equal('P30D');
     });
 
+  it('should get a restriction by id', async function() {
+    const id = await generateId();
+    await restrictions.insert({
+      restriction: {
+        id,
+        zone: ZONES.TWO,
+        resource: RESOURCES.STRAWBERRY,
+        method: 'limitOverDuration',
+        methodOptions: {
+          limit: 1,
+          duration: 'P30D'
+        }
+      }
+    });
+    const getRestriction = await restrictions.get({id});
+    const expectedRestriction = {
+      id,
+      zone: ZONES.TWO,
+      resource: RESOURCES.STRAWBERRY,
+      method: 'limitOverDuration',
+      methodOptions: {
+        limit: 1,
+        duration: 'P30D'
+      }
+    };
+    should.exist(getRestriction);
+    should.exist(getRestriction.meta);
+    should.exist(getRestriction.restriction);
+    should.exist(getRestriction.restriction.id);
+    expectedRestriction.id = getRestriction.restriction.id;
```

Comment:
these assertions can be covered with the deep.equal below with the suggestion above.  Also, `eql` is shorthand for `deep.equal`.